source/docs/software/driverstation/driver-station-labview-dashboard.rst

```diff
@@ -0,0 +1,127 @@
+FRC Driver Station LabVIEW Dashboard
+====================================
+
+The Dashboard application installed and launched by the FRC Driver Station is a LabVIEW program designed to provide teams with basic feedback from their robot, with the ability to expand and customize the information to suit their needs. This Dashboard application uses Network Tables and contains a variety of tools that teams may find useful.
+
+LabVIEW Dashboard
+-----------------
+
+.. image:: images/driver-station-labview-dashboard/labview-dashboard.png
+
+The Dashboard is broken into two main sections. The left pane is for displaying a camera image. The right pane contains:
+
+- Drive tab that contains indicators for joystick and drive motor values (hooked up by default when used with LabVIEW robot code), a gyro indicator, an Autonomous selection text box, a connection indicator and some controls and indicators for the camera,
+- Basic tab that contains some default controls and indicators,
+- Custom tab for customizing the dashboard using LabVIEW
+- Test tab for use with Test Mode in the LabVIEW framework
+- Commands tab for use with the new LabVIEW C&C Framework
+- Checklist tab that can be used to create task lists to complete before and/or between matches
+- Variables tab that displays the raw Network Tables variables in a tree view format
+
+The LabVIEW Dashboard also includes Record/Playback functionality, located in the bottom right. More detail about this feature is included below under `Record/Playback`_.
+
+Camera Image and Controls
+-------------------------
+
+.. image:: images/driver-station-labview-dashboard/camera-image-and-controls.png
```

Comment:
Update picture for 2020

Reply:
When updating the picture I noticed that there is no bandwidth indicator or framerate on the dashboard when no robot is connected. Is this correct and if so, should the notes about the bandwidth indicator and framerate be removed? Also, does the image below work?
![image](https://user-images.githubusercontent.com/15172054/81517037-7752ac80-92ee-11ea-8594-d0ef577fa0f5.png)

Reply:
When there is a camera connected, the bandwidth indicator does appear, so the description should remain. 
![image](https://user-images.githubusercontent.com/4885091/82153839-d5b3e980-981e-11ea-99be-70fc831533b7.png)

Reply:
Thanks for the clarification. Could I use this screenshot (as I don't have a roboRIO nearby) and could you send another with a camera connected to the camera tab (on the right pane)?

Reply:
Here's one with the camera on the right

![image](https://user-images.githubusercontent.com/4885091/82167276-2142b380-9870-11ea-8fbe-d35c321d19ac.png)

Reply:
Thanks. I find it odd that there is not a framerate or bandwidth indicator in the bottom bar below the camera tab.

Reply:
The bandwidth indicator is a combined bandwidth. The framerate indicator is by the bandwidth indicator. 
![image](https://user-images.githubusercontent.com/4885091/82170709-1725b280-987a-11ea-825c-fa0759317f8d.png)

Reply:
That makes a lot of sense. I'll include a note in the camera tab section. Thanks!